src/client/components/dimension-tile/dimension-tile.tsx

```diff
@@ -183,7 +183,7 @@ export class DimensionTile extends React.Component<DimensionTileProps, Dimension
     }
 
     if (sortOn.measure) {
-      query = query.performAction(sortOn.measure.toApplyExpression());
+      query = query.performAction(sortOn.measure.toApplyExpression(0));
```

Comment:
maybe this should be the default value 0 which is nonNested ?

Reply:
Good idea!